Assets/Scripts/MonoBehavior/EEffectType.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+
+public enum EEffectType 
+{
+	"BlackHole", 
+	"Bomb", 
+	"Coin", 
+    "DamageOverTime", 
+    "DamageReduction", 
+    "DamageToSelf", 
+    "Heal", 
+    "SlipperyTrail", 
+    "Slowness"
```

Comment:
O enum EEffectType existe pra definir os efeitos pra status effects, logo só eles são relevantes de existir aqui. Reduz os EEffectTypes pra

- DamageModifier (Isso vai modificar a porcentagem do dano visando reduzir ou aumentar ela)
- Slow (Reduz a velocidade de movimento)
- Weaken (Reduz a velocidade do carregamento de força do flick)